### ISSUE TITLE
Can't mine and can't compile too much errors

### ISSUE THREAD

**Issue by mrschaeffer2018** [zcoinofficial/ccminer#72]
Hi, can you guys create a cuda 9.0 release for linux 16 because i have only libcudart9.0 and an old SM 3.7 compute capability i tried to edit the makefile.in to add 3.7 arch and remove the 7.5 arch that do not work with cuda 9.0 but the lyra2/lyra2z.cu trigger some error as 3.7 was not intended when was compiled and i couldn't figure it out how to change or remove the checks for arch in that file so can u guys create a cuda 9.0 for 3.7 compute capabilities card release removing from the code the need for newer like 7.5 arch?

**Comment by djm34** [comment]
hmm... I am pretty sure that there is already binaries for cuda 9 in the release area... oh cuda 9.2 and you want cuda 9.0...

there is a very easy fix to your problems: Upgrade your nvidia drivers... 
there hasn't been a good  reason to use cuda 9.0 since at least 4 years... so upgrade your drivers, whatever card you are using will still be supported by more recent cuda version

**Comment by djm34** [comment]
what card are you using by the way ? For info mtp requires at least 4.4Gb of vram

**Comment by mrschaeffer2018** [comment]
I'm using the institute that i work k80 the people there are ok with that, they are idle most of the time but i can't upgrade the CUDA yet they have old stuff running and would require a protocol if i paste the log of error in the lyra2/lyra2z.cu file  that prevents me to compile probably because of older 3.7 SM do you believe you can help me with some tweaks? Maybe removing the parts that says about newer cards as i will not gonna use it.

**Comment by djm34** [comment]
I'll have a look later in the day, that lyra2/2z version wasn't written by me... and is a huge pain in the way it was written (unnecessary complication to work for various platform). 

Does the k80 has enough memory ?
To be honest, the performance on kepler won't be great as it is a rather old architecture

**Comment by mrschaeffer2018** [comment]
Yes the k80 have 12gb i don't know about performance but as long as i have a lot of them in idle availble by quantity it can compensate it.

**Comment by djm34** [comment]
sorry, I'll try to have a look soon. what is the compute version of the kepler ?

**Comment by mrschaeffer2018** [comment]
The compute version it is 37, thanks for responding and for the effort in help, thank you.

**Comment by mrschaeffer2018** [comment]
Hi, just updating the issue, i was able to start the miner downloading another libcudart-9.2 and pointing to that lib when executing the miner, but when started i get this error below, do you know what coud it be?

[2021-02-22 09:52:55] POOL 0: firo.mintpond.com:3000 USER a7ix3YSPkfgH2EyCg9vzsL4A6CSckLeSot.maninho -s 30
[2021-02-22 09:52:55] Starting on stratum+tcp://firo.mintpond.com:3000
[2021-02-22 09:52:55] 1 miner thread started, using 'mtp' algorithm.
h[2021-02-22 09:52:58] MESSAGE FROM SERVER: Strict mode enabled
[2021-02-22 09:52:58] MESSAGE FROM SERVER: Verbose level is high.
[2021-02-22 09:52:58] MESSAGE FROM SERVER: Static difficulty (d) is not available.
[2021-02-22 09:52:58] MESSAGE FROM SERVER: Difficulty change: 2048 (0.03125)
[2021-02-22 09:52:58] MESSAGE FROM SERVER: New Job: 00006372
[2021-02-22 09:52:58] MESSAGE FROM SERVER: Welcome to the North American Firo stratum.
[2021-02-22 09:52:59] Stratum difficulty set to 0.03125
[2021-02-22 09:52:59] GPU #0: Intensity set to 21.0156, 2129920 cuda threads number of multiproc 13
[2021-02-22 09:52:59] MESSAGE FROM SERVER: Optimizing difficulty may take several minutes.
Cuda error in func 'mtp_setBlockTarget' at line 807 : invalid device symbol.

Would be that due to using CUDA 9.0 not 9.2 ?
